addons/pos_self_order/__manifest__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 {
     "name": "POS Self Order",
-    'version': '1.0.1',
+    'version': '1.0',
```

Comment:
Better increase the version number.
```suggestion
    'version': '1.1',
```